app/models/async_transaction/vet360/base.rb

```diff
@@ -2,6 +2,17 @@
 
 module AsyncTransaction
   module Vet360
-    class Base < AsyncTransaction::Base; end
+    class Base < AsyncTransaction::Base
+      def self.start(user, response)
+        create(
+          user_uuid: user.uuid,
+          source_id: user.vet360_id,
+          source: 'vet360',
+          status: 'sent',
```

Comment:
@rlbaker @emersonthis - Took a stab at this, but do we want this initial status to read `"sent"`?  If no, pls recommend an initial status?

P.S.  We should really create all of our anticipated statuses as constants, and add them to this file.

Reply:
Agreed. Status constants. FWIW 'sent' feels right to me.

Reply:
Cool.  I'll create one for now, as to not block the shipping of this PR.  We can circle back with others as we move forward.